rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -284,10 +284,10 @@ $$("#${detailsId}").bind( "show", function(event, ui) {
     val jsId = JsNodeId(sm.node.main.id,salt)
     displayTabFilesystems(jsId, sm) ++
     displayTabNetworks(jsId, sm) ++
-    node.map(displayTabProperties(jsId, _)).getOrElse(Nil) ++
     displayTabVariable(jsId, sm) ++
     displayTabProcess(jsId, sm) ++
     displayTabVM(jsId, sm) ++
+    node.map(displayTabProperties(jsId, _)).getOrElse(Nil) ++
```

Comment:
I moved this line because I moved the order of tabs above, but I suspect it's not necessary. If it could be dangerous, and is not necessary, just let me know and I'll put it back.